targets/TARGET_STM/TARGET_STM32F4/TARGET_STM32F437xG/device/TOOLCHAIN_IAR/stm32f437xx.icf

```diff
@@ -15,8 +17,8 @@ define symbol __ICFEDIT_region_CCMRAM_end__    = 0x1000FFFF;
 define symbol __ICFEDIT_region_BKPSRAM_start__ = 0x40024000;
 define symbol __ICFEDIT_region_BKPSRAM_end__   = 0x40024FFF;
 /*-Sizes-*/
-/*Heap 1/2 of ram and ISR stack 4 kbytes*/
-define symbol __ICFEDIT_size_cstack__ = 0x1000;
+/*Heap 1/2 of ram and stack 1/8*/
+define symbol __ICFEDIT_size_cstack__ = 0x6000;
```

Comment:
Oops, well spotted, that was a hangover from a previous version of this PR; we had reduced the IAR stack size afterwards, now corrected/re-based/re-pushed.

Test results (with this latest rebase):
```
+-------------------------+-----------------+-------------------------------------+----------------------------------+--------+--------+--------+--------------------+
| target                  | platform_name   | test suite                          | test case                        | passed | failed | result | elapsed_time (sec) |
+-------------------------+-----------------+-------------------------------------+----------------------------------+--------+--------+--------+--------------------+
| UBLOX_C030_U201-GCC_ARM | UBLOX_C030_U201 | mbed-os-tests-mbed_drivers-flashiap | FlashIAP - init                  | 1      | 0      | OK     | 0.04               |
| UBLOX_C030_U201-GCC_ARM | UBLOX_C030_U201 | mbed-os-tests-mbed_drivers-flashiap | FlashIAP - program               | 1      | 0      | OK     | 3.06               |
| UBLOX_C030_U201-GCC_ARM | UBLOX_C030_U201 | mbed-os-tests-mbed_drivers-flashiap | FlashIAP - program errors        | 1      | 0      | OK     | 0.05               |
+-------------------------+-----------------+-------------------------------------+----------------------------------+--------+--------+--------+--------------------+
| UBLOX_C030_U201-GCC_ARM | UBLOX_C030_U201 | mbed-os-tests-mbed_hal-flash        | Flash - buffer alignment test    | 1      | 0      | OK     | 2.02               |
| UBLOX_C030_U201-GCC_ARM | UBLOX_C030_U201 | mbed-os-tests-mbed_hal-flash        | Flash - clock and cache test     | 1      | 0      | OK     | 0.1                |
| UBLOX_C030_U201-GCC_ARM | UBLOX_C030_U201 | mbed-os-tests-mbed_hal-flash        | Flash - erase sector             | 1      | 0      | OK     | 1.05               |
| UBLOX_C030_U201-GCC_ARM | UBLOX_C030_U201 | mbed-os-tests-mbed_hal-flash        | Flash - init                     | 1      | 0      | OK     | 0.08               |
| UBLOX_C030_U201-GCC_ARM | UBLOX_C030_U201 | mbed-os-tests-mbed_hal-flash        | Flash - mapping alignment        | 1      | 0      | OK     | 0.05               |
| UBLOX_C030_U201-GCC_ARM | UBLOX_C030_U201 | mbed-os-tests-mbed_hal-flash        | Flash - program page             | 1      | 0      | OK     | 2.53               |
+-------------------------+-----------------+-------------------------------------+----------------------------------+--------+--------+--------+--------------------+
| UBLOX_C030_U201-IAR     | UBLOX_C030_U201 | mbed-os-tests-mbed_drivers-flashiap | FlashIAP - init                  | 1      | 0      | OK     | 0.04               |
| UBLOX_C030_U201-IAR     | UBLOX_C030_U201 | mbed-os-tests-mbed_drivers-flashiap | FlashIAP - program               | 1      | 0      | OK     | 3.04               |
| UBLOX_C030_U201-IAR     | UBLOX_C030_U201 | mbed-os-tests-mbed_drivers-flashiap | FlashIAP - program errors        | 1      | 0      | OK     | 0.05               |
+-------------------------+-----------------+-------------------------------------+----------------------------------+--------+--------+--------+--------------------+
| UBLOX_C030_U201-IAR     | UBLOX_C030_U201 | mbed-os-tests-mbed_hal-flash        | Flash - buffer alignment test    | 1      | 0      | OK     | 2.09               |
| UBLOX_C030_U201-IAR     | UBLOX_C030_U201 | mbed-os-tests-mbed_hal-flash        | Flash - clock and cache test     | 1      | 0      | OK     | 0.1                |
| UBLOX_C030_U201-IAR     | UBLOX_C030_U201 | mbed-os-tests-mbed_hal-flash        | Flash - erase sector             | 1      | 0      | OK     | 1.14               |
| UBLOX_C030_U201-IAR     | UBLOX_C030_U201 | mbed-os-tests-mbed_hal-flash        | Flash - init                     | 1      | 0      | OK     | 0.09               |
| UBLOX_C030_U201-IAR     | UBLOX_C030_U201 | mbed-os-tests-mbed_hal-flash        | Flash - mapping alignment        | 1      | 0      | OK     | 0.06               |
| UBLOX_C030_U201-IAR     | UBLOX_C030_U201 | mbed-os-tests-mbed_hal-flash        | Flash - program page             | 1      | 0      | OK     | 2.7                |
+-------------------------+-----------------+-------------------------------------+----------------------------------+--------+--------+--------+--------------------+
| UBLOX_C030_U201-ARM     | UBLOX_C030_U201 | mbed-os-tests-mbed_drivers-flashiap | FlashIAP - init                  | 1      | 0      | OK     | 0.04               |
| UBLOX_C030_U201-ARM     | UBLOX_C030_U201 | mbed-os-tests-mbed_drivers-flashiap | FlashIAP - program               | 1      | 0      | OK     | 2.98               |
| UBLOX_C030_U201-ARM     | UBLOX_C030_U201 | mbed-os-tests-mbed_drivers-flashiap | FlashIAP - program errors        | 1      | 0      | OK     | 0.05               |
+-------------------------+-----------------+-------------------------------------+----------------------------------+--------+--------+--------+--------------------+
| UBLOX_C030_U201-ARM     | UBLOX_C030_U201 | mbed-os-tests-mbed_hal-flash        | Flash - buffer alignment test    | 1      | 0      | OK     | 2.03               |
| UBLOX_C030_U201-ARM     | UBLOX_C030_U201 | mbed-os-tests-mbed_hal-flash        | Flash - clock and cache test     | 1      | 0      | OK     | 0.07               |
| UBLOX_C030_U201-ARM     | UBLOX_C030_U201 | mbed-os-tests-mbed_hal-flash        | Flash - erase sector             | 1      | 0      | OK     | 1.05               |
| UBLOX_C030_U201-ARM     | UBLOX_C030_U201 | mbed-os-tests-mbed_hal-flash        | Flash - init                     | 1      | 0      | OK     | 0.06               |
| UBLOX_C030_U201-ARM     | UBLOX_C030_U201 | mbed-os-tests-mbed_hal-flash        | Flash - mapping alignment        | 1      | 0      | OK     | 0.05               |
| UBLOX_C030_U201-ARM     | UBLOX_C030_U201 | mbed-os-tests-mbed_hal-flash        | Flash - program page             | 1      | 0      | OK     | 2.53               |
+-------------------------+-----------------+-------------------------------------+----------------------------------+--------+--------+--------+--------------------+
```